src/entry.js

```diff
@@ -1,40 +1,16 @@
-const glob = require('glob');
-const shared = require('./shared');
-
 function StacheEntryPlugin() {
-  let _pluginPaths = glob.sync(
-    './*.js',
-    {
-      ignore: [
-        './*.spec.js',
-        './entry.js',
-        './shared.js'
-      ],
-      cwd: __dirname
-    }
-  );
-
   const preload = (content, resourcePath, skyPagesConfig) => {
-    if (!_pluginPaths.length) {
-      return content;
-    }
-
-    _pluginPaths.forEach(pluginPath => {
-      try {
-        const plugin = require(pluginPath);
-
-        if (!plugin.preload) {
-          return;
-        }
-
-        const altered = plugin.preload(content, resourcePath, skyPagesConfig);
-
-        if (content !== altered) {
-          content = altered;
-        }
-      } catch (error) {
-        throw new shared.StachePluginError(error.message);
```

Comment:
The `try/catch` block is necessary because it will alert the developer of a specific error in any of the individual plugins. (Since we're just doing an uncatchable `require`.)

```
const preload = (content, resourcePath, skyPagesConfig) => {
  try {
    const plugins = [
      require('./config'),
      require('./include'),
      require('./code-block'),
      require('./json-data'),
      require('./route-metadata'),
      require('./template-reference-variable')
    ];

    plugins.forEach(plugin => {
      content = plugin.preload(content, resourcePath, skyPagesConfig);
    });
  } catch (error) {		
    throw new shared.StachePluginError(error.message);
  }

  return content;
};
```